src/libs/validationHandler.ts

```diff
@@ -0,0 +1,65 @@
+import { Request, Response, NextFunction } from 'express';
+export default (config) => (req: Request,res: Response,next: NextFunction) => {
+    const errors = [];
+    Object.keys(config).forEach((key) => {
+        const i = 0;
+        const keys = config[key];
+        const values=keys.in.map(location => {
+            return req[location][key]
+        });
+         
+        let request = values.find(val=>{return isNull(val)})
+        const regex = keys.regex;
+        if ((!keys.required) && !(isNull (request))) {
+            return request = keys.default;
+        }
+        if  ((keys.number) && !(isNaN(Number(request))))
```

Comment:
Remove remove "!" from  `!(isNaN(Number(request)`.  isNaN() will return true for non number value.